install/infra/modules/tools/cloud-dns-ns/main.tf

```diff
@@ -1,17 +1,20 @@
-variable credentials {}
-variable nameservers {}
-variable domain_name {}
-variable managed_dns_zone {}
-variable dns_project {}
+variable "nameservers" {}
+variable "domain_name" {}
+variable "managed_dns_zone" {}
+variable "dns_project" {}
+variable "credentials" {}
 
 provider "google" {
+  alias       = "dns"
+  project     = "dns-for-playgrounds"
```

Comment:
**issue(non-blocking):** If we're anticipating that external users will be consuming our modules then we can't hardcode the names of our projects in `install/infra/modules/*`.

Would the following work? (Not sure if it's even valid syntax, just shooting from the hip)

```suggestion
  project     = var.dns_project
```

Non-blocker, feel free to skip this issue and address later.

Reply:
Will take this change up on the [corresponding PR](https://github.com/gitpod-io/gitpod/pull/13541)